### DISCUSSION TITLE
Syncing Slicers

### DISCUSSION THREAD

**Discussion by jcaravella** [nbrown02/FlowViz#46] [Q&A]
FlowViz is incredible. I've been trying to add flow metrics to my org for some time and the panels and details in this report are amazing.

I've tried to sync slicers across tabs, and it works fine in the Desktop, but when I publish the slicers don't sync and I cannot figure out why. I've made sure the slicers aren't duplicated with one behind the other.  Any idea why?

**Comment by nbrown02** [comment]
Hey @jcaravella - thank you so much. If you could leave feedback on the [AzDO marketplace review page](https://marketplace.visualstudio.com/items?itemName=agile-extensions.flowviz&ssr=false) that would be even better :)

I'll have a look into this for you to see if I can replicate/advise - I don't use web too often but can see what I can do. Is it using the 'project' template?

**Reply by jcaravella** [reply]
I will leave feedback. I'm using the Org template.
